src/lib/client/localStorage/backup.ts

```diff
@@ -12,10 +12,17 @@ export const loadBackup = (
 ): { profile: Profile; keys: Keys } => {
   const { profile, keys } = JSON.parse(backup);
 
+  // TODO: temporary fix for missing allowsAnalytics
```

Comment:
@AndrewCLu  

![Screenshot 2024-02-10 at 12 53 35](https://github.com/nfc-denver/nfc-denver/assets/20865711/473ed0a5-a676-450d-bdb3-244e2acb0bd7)

Reply:
This change is made during this previous PR [here](https://github.com/nfc-denver/nfc-denver/commit/c6f5b86d72a27bbd4738b77c09f1d50a43f89162). You just need to delete your user from local db and register a new account, no need for this work around.